sdk/communication/communication-identity/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.2.0 (Unreleased)
+
+### Features Added
+- Added support to customize the Communication Identity access token's validity period:
+    - Added optional parameter that provide the ability to create a Communication Identity access token with custom expiration:
+        - `createUserAndToken(scopes: TokenScope[], tokenExpirationInMinutesOrOptions: number | OperationOptions = {}, options: OperationOptions = {})`
```

Comment:
fix it regarding the latest API change same for the line 9

Reply:
Updated.